app/routes/labs/route.tsx

```diff
@@ -25,10 +25,7 @@ export default function () {
   return (
     <>
       <Header />
-      <h1>ACROSS</h1>
-      <main>
-        <Outlet />
-      </main>
```

Comment:
Why did you remove the main section?

Reply:
I didn't see an equivalent main section in other index pages. Including one also added a white border below the header.

Reply:
The reason that this page has a `<main>` element and most others don't is that this is the layout route for all pages under /labs. The `<main>` element is required for accessibility. The white border is a problem; I will do some research to determine how to tell the browser to prefer dark mode, which is supported by USWDS.